dpub-aam/index.html

```diff
@@ -252,7 +248,7 @@ <h2>Role mapping</h2>
 
       <p>
         Platform <a class="termref">accessibility <abbr title="Application Programming Interfaces">APIs</abbr></a> traditionally have had a finite set of predefined <a class="termref">roles</a> that
-        are expected by <a class="termref">assistive technologies</a> on that platform and only one or two roles are usually exposed. In contrast,
+        are expected by <a class="termref">assistive technologies</a> on that platform and only one or two roles may be exposed. In contrast,
```

Comment:
This line change reverts a change from https://github.com/w3c/aria/pull/2488/ which specifically addressed review concerns regarding normative language

Reply:
Thanks for catching this @pkra Should be good now